db/postgres/modtags.go

```diff
@@ -0,0 +1,56 @@
+package postgres
+
+import (
+	"context"
+	"errors"
+
+	"github.com/patrickmn/go-cache"
+)
+
+func CreateModTag(modTag *ModTag, ctx *context.Context) (*ModTag, error) {
+	if len(modTag.Name) > 20 {
+		return nil, errors.New("Mod tag name is more than 20 characters")
+	}
+
+	exists := GetModTagByName(modTag.Name, ctx)
+	if exists != nil {
+		return nil, errors.New("Mod tag already exists")
+	}
```

Comment:
You don't need to check if it exists, try inserting it and see if you got an error in response

Reply:
Got it